.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+<!-- Thank you for contributing to Express! -->
+
+### Description
+<!-- Provide a brief description of your changes -->
+
+### Type of Change
+<!-- Put an `x` in all the boxes that apply -->
+
+- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
+- [ ] 🚀 New feature (non-breaking change which adds functionality)
+- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] 📝 Documentation update
+- [ ] 🎨 Style update (formatting, local variables)
+- [ ] 🔨 Refactoring (no functional changes, no API changes)
+- [ ] 🔧 Build or dependency related changes
+
+### How Has This Been Tested?
+<!-- Please describe the tests you've added or modified -->
+
+- [ ] Added/modified unit tests
+- [ ] Added/modified integration tests
+- [ ] Manual testing
+
+### Checklist
+<!-- Put an `x` in all the boxes that apply -->
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/expressjs/express/blob/master/Contributing.md) document
+- [ ] My code follows the code style of this project
+- [ ] I have updated the documentation accordingly
+- [ ] I have added tests to cover my changes
+- [ ] All new and existing tests passed
+- [ ] My commits follow [conventional commits](https://www.conventionalcommits.org/) format
```

Comment:
while I love the idea of following conventional commits, maybe we should agree on it first.